lib/wrapper.js

```diff
@@ -57,14 +59,15 @@ class RequestWrapper {
   }
 
   _handleResponseError(error) {
-    logger.error('fatal_error', error.message, this._getLogParameters());
+    logger.fromError('fatal_error', error, this._getLogParameters());
```

Comment:
fatal_error doesn't really tell anything
can we rename to like response-handling or something that tells where the log happened?

Reply:
It is not up to us. Let's see what @blacksonic says about it.

Reply:
The log namespace describes the package, if this string is unique inside the package it will tell where it happened.
`logger.fromError` also stores the stack trace.
What else is needed?

Reply:
we can add a different name that is more descriptive

Reply:
I'm good either way, just suggesting, more explicit is better